packages/syft/src/syft/service/action/action_object.py

```diff
@@ -1969,30 +1970,30 @@ def _repr_markdown_(self, wrap_as_python: bool = True, indent: int = 0) -> str:
         elif not self.is_twin:
             res = "Pointer"
 
-        if isinstance(self.syft_action_data_cache, ActionDataEmpty):
+        if isinstance(self.syft_action_data, ActionDataEmpty):
```

Comment:
Not sure about these changes, not using the cache here changes the behaviour

'if the cache is empty use the cached repr, else get repr from action data'